server/src/dtos/asset.dto.ts

```diff
@@ -46,6 +46,9 @@ export class UpdateAssetBase {
   @IsLongitude()
   @IsNotEmpty()
   longitude?: number;
+
+  @Optional()
```

Comment:
should this assert the value is bigger than or equal to 0? also we need a validator to enforce it is a number.

also, I think this should allow "clearing" the rating by setting it to `null`.

Reply:
Thanks. Correct, I added three validators:   @IsInt()  @Max(5)  @Min(0)
The Exif library doesn't allow to "clear" the rating. Just set it to 0.

Reply:
You should definitely wrap those in code blocks `` (using backticks) to not ping random people lmao.

Reply:
Fixed for you.